lib/routes/account.js

```diff
@@ -1420,6 +1420,77 @@ module.exports = function (
       }
     },
     {
+      method: 'GET',
+      path: '/account/sessions',
+      config: {
+        auth: {
+          strategy: 'sessionToken'
+        },
+        response: {
+          schema: isA.array().items(isA.object({
+            id: isA.string().regex(HEX_STRING).required(),
+            lastAccessTime: isA.number().min(0).required().allow(null),
+            lastAccessTimeFormatted: isA.string().optional().allow(''),
+            userAgent: isA.string().max(255).required().allow(''),
+
+            deviceId: isA.string().regex(HEX_STRING).allow(null),
+            deviceName: isA.string().max(255).required().allow('').allow(null),
+            deviceType: isA.string().max(16).required().allow(null),
+            deviceCallbackURL: isA.string().uri({ scheme: 'https' }).max(255).optional().allow('').allow(null),
+            deviceCallbackPublicKey: isA.string().max(88).regex(URLSAFEBASE64).optional().allow('').allow(null),
+            deviceCallbackAuthKey: isA.string().max(24).regex(URLSAFEBASE64).optional().allow('').allow(null),
+
+            isDevice: isA.boolean().required(),
+            isCurrentDevice: isA.boolean().required()
+          }))
+        }
+      },
+      handler: function (request, reply) {
+        log.begin('Account.sessions', request)
+        var sessionToken = request.auth.credentials
+        var uid = sessionToken.uid
+
+        db.sessions(uid).then(
```

Comment:
Just a general question, noticed that this endpoint and some other `/account/devices` endpoints don't have rate limiting in place. Was there a reason for that? Depending on the number of sessions, I could see this request being kind of heavy since it has to join the devices table.